<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBox</class>
 <widget class="QDialog" name="AddBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Object Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Size (x, y and z): </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="G">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RGB Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sizeX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9990.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="B">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="addBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="R">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sizeY">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9990.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sizeZ">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9990.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
